node/narwhal/src/helpers/committee.rs

```diff
@@ -169,7 +169,7 @@ impl<N: Network> Committee<N> {
     /// Note: This ensures the method returns a deterministic result that is SNARK-friendly.
     fn sorted_members(&self) -> indexmap::map::IntoIter<Address<N>, u64> {
         let members = self.members.clone();
-        members.sorted_by(|address1, stake1, address2, stake2| {
+        members.sorted_unstable_by(|address1, stake1, address2, stake2| {
```

Comment:
Is this safe? Can you provide more context on the implications (I understand what `unstable` means here)

Reply:
unstable sorting is just faster, so it's preferable if the order of equal elements doesn't matter; in our case, the `members` are initially sorted by insertion order and already deduplicated by the `Address` (since it's the key used for hashing), so unless we could have identical X coordinates _and_ would still want to retain the insertion order, we are free to do unstable sorting.